### ISSUE TITLE
ERROR: could not unmarshal event: unknown PrivateNetworkRequestPolicy value

### ISSUE THREAD

**Issue by JohnnyTing** [chromedp/chromedp#797]
<pre>
$ github.com/chromedp/chromedp v0.6.5

$ go version go1.13.4
</pre>
```go

opts := []chromedp.ExecAllocatorOption{
		chromedp.ExecPath(constants.ExecPath),
		chromedp.UserAgent(constants.UserAgent),
		chromedp.NoFirstRun,
		chromedp.NoDefaultBrowserCheck,
		//chromedp.Headless,
		chromedp.DisableGPU,
	}

	ctx, cancel := chromedp.NewExecAllocator(context.Background(), opts...)
	defer cancel()

	ctx, cancel = chromedp.NewContext(ctx)
	defer cancel()

	err = chromedp.Run(ctx, chromedp.Tasks{
		network.Enable(),
		chromedp.Navigate(link),
	})
```

appear below error:

ERROR: could not unmarshal event: unknown PrivateNetworkRequestPolicy value

i can't find the error reason, help me...

**Comment by ZekeLu** [comment]
Please provide the version information of the `github.com/chromedp/cdproto` package and the browser. According to the error message, It looks like that the event contains PrivateNetworkRequestPolicy (sent by the browser) is not recognized by `github.com/chromedp/cdproto`.

**Comment by JohnnyTing** [comment]
> github.com/chromedp/cdproto
thanks for your answer,  
```shell
~ go list -m github.com/chromedp/cdproto
github.com/chromedp/cdproto v0.0.0-20210126020047-7ec7357d1463

```

**Comment by JohnnyTing** [comment]
@ZekeLu thank you very much.  it works good when update chromedb latest version.

**Comment by ZekeLu** [comment]
Great! The value `WarnFromInsecureToMorePrivate`  is added in `
v0.0.0-20210211211551-bf465a41a1f9`. I guesses  this is the reason.

```diff
diff --git a/network/types.go b/network/types.go
index ef2a218..d31595e 100644
--- a/network/types.go
+++ b/network/types.go
@@ -1343,6 +1407,7 @@ func (t PrivateNetworkRequestPolicy) String() string {
 const (
    PrivateNetworkRequestPolicyAllow                          PrivateNetworkRequestPolicy = "Allow"
    PrivateNetworkRequestPolicyBlockFromInsecureToMorePrivate PrivateNetworkRequestPolicy = "BlockFromInsecureToMorePrivate"
+   PrivateNetworkRequestPolicyWarnFromInsecureToMorePrivate  PrivateNetworkRequestPolicy = "WarnFromInsecureToMorePrivate"
 )
```

**Comment by shelld3v** [comment]
Hi @ZekeLu, I am using the latest version of chromedp, but still the same issue

**Comment by shelld3v** [comment]
> Hi @ZekeLu, I am using the latest version of chromedp, but still the same issue

Fixed

**Comment by jcamaproper** [comment]
> 

Hi, I have the same issue, How did you fix that?

**Comment by sifr0x** [comment]
@jcamaproper you can fix this by upgrading chromedp to a newer version.

**Comment by jvegaseg** [comment]
It seams that with latest Google Chrome version (98.0.4758.80) the error reappears again.
I downgraded to 97.0.4692.99 and the error disappear.

**Comment by ZekeLu** [comment]
@jvegaseg Yes, Google Chrome introduced two more preflight policies (`PreflightWarn` and `PreflighBlock`) recently (see https://chromium.googlesource.com/chromium/src/+/3ad823199a84a5d0574fc332701d5c549b10de9d). `v0.7.7` has just been released to address this issue.

Thanks!

**Comment by jvegaseg** [comment]
Thanks, solved after upgrading to v0.7.7
